File: app/src/main/java/com/norvera/confession/data/dao/PrayersDao.java
package com.norvera.confession.data.dao;

import com.norvera.confession.data.models.PrayersEntry;

import java.util.List;

import androidx.lifecycle.LiveData;
import androidx.room.Dao;
import androidx.room.Query;

@Dao
public abstract class PrayersDao {
    @Query("SELECT * FROM  PRAYERS")
    public abstract LiveData<List<PrayersEntry>> loadAllPrayers();
}

File: app/src/main/java/com/norvera/confession/data/dao/PrayersDao.kt
package com.norvera.confession.data.dao

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Query
import com.norvera.confession.data.models.PrayersEntry

@Dao
abstract class PrayersDao {
    @Query("SELECT * FROM  PRAYERS")
    abstract fun loadAllPrayers(): LiveData<List<PrayersEntry>>
}
